main/utils/textEntry.c

```diff
@@ -196,10 +206,8 @@ bool textEntryDraw(int64_t elapsedUs)
     }
     // Draw an indicator for the current key modifier
     _drawTypeMode();
-    // Draw the keyboard
-    int16_t keyboardH = _drawKeyboard();
-    // Draw the currently typed string
-    _drawStr(keyboardH, elapsedUs);
+    // Draw the rest of the fucking owl
```

Comment:
Love the meme, let's keep it PG.
```suggestion
    // Draw the rest of the owl
```

Reply:
Oh dang, I thought I deleted that.